botnim/cli.py

```diff
@@ -53,68 +53,65 @@ def reverse_lines(text: str) -> str:
         click.echo(f"Error: {str(e)}", err=True)
-        raise click.Abort()
 
 @query_group.command(name='list-indexes')
-@click.argument('environment', type=click.Choice(['production', 'staging']))
-@click.option('--bot', type=click.Choice(AVAILABLE_BOTS), default='takanon', 
-              help='Bot to list indexes for')
+@click.argument('environment', type=click.Choice(VALID_ENVIRONMENTS))
+@click.option('--bot', type=click.STRING, default=None, help='Filter indexes by bot name')
 @click.option('--rtl', is_flag=True, help='Display results in right-to-left order')
 def list_indexes(environment: str, bot: str, rtl: bool):
     """List all available indexes in the vector store."""
     try:
         indexes = get_available_indexes(environment, bot)
-        click.echo("Available indexes:")
         for index in indexes:
-            index_display = index[::-1] if rtl else index
-            click.echo(f"  - {mirror_brackets(index_display)}")
+            if rtl:
+                index = index[::-1]
+            click.echo(index)
     except Exception as e:
         click.echo(f"Error: {str(e)}", err=True)
-        raise click.Abort()
 
 @query_group.command(name='show-fields')
-@click.argument('environment', type=click.Choice(['production', 'staging']))
-@click.argument('bot', type=click.Choice(AVAILABLE_BOTS), default='takanon')
+@click.argument('environment', type=click.Choice(VALID_ENVIRONMENTS))
+@click.argument('bot', type=click.STRING)
 @click.argument('context', type=click.STRING)
 @click.option('--rtl', is_flag=True, help='Display results in right-to-left order')
 def show_fields(environment: str, bot: str, context: str, rtl: bool):
-    """Show all available fields in the index."""
+    """Show the fields/structure of an index."""
     try:
         mapping = get_index_fields(environment, bot, context)
-        formatted_mapping = format_mapping(mapping)
+        formatted = format_mapping(mapping)
         if rtl:
-            formatted_mapping = reverse_lines(mirror_brackets(formatted_mapping))
-        click.echo(f"\nFields in index for bot '{bot}', context '{context}':")
-        click.echo(formatted_mapping)
+            formatted = reverse_lines(formatted)
+        click.echo(formatted)
     except Exception as e:
         click.echo(f"Error: {str(e)}", err=True)
-        raise click.Abort()
 
 @cli.command(name='assistant')
-@click.option('--assistant-id', type=str, help='ID of the assistant to chat with')
-@click.option('--openapi-spec', type=str, default='budgetkey', help='either "budgetkey" or "takanon"')
-@click.option('--rtl', is_flag=True, help='Enable RTL support for Hebrew/Arabic')
-def assistant(assistant_id, openapi_spec, rtl):
-    """Start an interactive chat with an OpenAI assistant."""
-    assistant_main(assistant_id, openapi_spec, rtl)
+@click.option('--assistant-id', type=click.STRING, help='Assistant ID to use')
+@click.option('--openapi-spec', type=click.STRING, default=None, help='OpenAPI spec to use')
```

Comment:
Why change the documentation to be less useful?

Reply:
fixed

Reply:
not fixed yet

Reply:
reverted, but TBH, I don't think this is better.